### PR TITLE
BigtableIO StreamingWrite Setup and Teardown

### PR DESCRIPTION
**Pull request by ianzhou1**

### CODE REVIEW

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableService.java [hunk 1]
```diff
@@ -52,6 +52,13 @@
...
+     * Flushes the writer.
+     *
+     * @throws IOException if any writes did not succeed
+     */
+    void flush() throws IOException;
+
+    /**
      * Closes the writer.
      *
      * @throws IOException if any writes did not succeed
```
Comment (dhalperi) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableService.java]: fix javadoc -- if there is an error closing the writer.

**Comment by dhalperi** [comment]
R: @vikkyrk

**Comment by dhalperi** [comment]
Prediction: BigtableWriteIT will break because Dataflow service does not support this feature yet. This is probably okay.

**Comment by vikkyrk** [comment]
Also looks like there new methods are not supported in DoFnTester? If you have any tests using it, then you probably need to explicitly call setup/teardown?

**Comment by dhalperi** [comment]
@vikkyrk good point -- file a JIRA?

**Comment by vikkyrk** [comment]
@dhalperi filed https://issues.apache.org/jira/browse/BEAM-559. Assigned to you, redirect appropriately :)

**Comment by dhalperi** [comment]
LGTM, merged

**Pull request closed without merging**